.github/workflows/deploy-mainnet.yml

```diff
@@ -0,0 +1,166 @@
+---
```

Comment:
the `deploy-calibnet.yml` and `deploy-mainnet.yml` seem almost the same. Seems it's quite prone errors to manually create them. Should we have a single template for those and automatically generate them?

Reply:
Hmm, good thought - this is possible I think. Let me look into it